guides/common/modules/proc_configuring-hammer-cli-to-accept-freeipa-credentials.adoc

```diff
@@ -11,7 +11,7 @@ For more information, see xref:configuring-the-freeipa-authentication-source-on-
 * Open the `~/.hammer/cli.modules.d/foreman.yml` file on your {ProjectServer} and update the list of `foreman` parameters:
 ** To enforce session usage, enable `:use_sessions:`:
 +
-[options="nowrap", subs="+quotes,verbatim,attributes"]
+[source, none, options="nowrap", subs="+quotes,verbatim,attributes"]
```

Comment:
Why not syntax highlight this as YAML, which it is.

Reply:
Applied to both files.